cypress/integration/models/CreateModel.AllEntityTypes.js

```diff
@@ -0,0 +1,23 @@
+/**
+* Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+*/
+
+const helpers = require('../../support/helpers.js')
+const models = require('../../support/Models')
+const entities = require('../../support/Entities')
+
+describe('Create All Entity Types', () =>
+{
+  after(() => { cy.VerifyMonitorFinds() })
```

Comment:
Is this your function or cy?   If yours I might give it a more descriptive name or a comment explaining what it does

Reply:
Added comment to the main method, and removed all the after() functions.
This is used to help track down rare situations that the monitor finds.